dragonfly_doe2/doe/utils.py

```diff
@@ -26,6 +26,7 @@ def lower_left_properties(room_2d):
     """Get the vertices, boundary conditions and windows starting from lower left.
     v2 WIP
     """
+    room_2d.remove_duplicate_vertices
```

Comment:
Isn't this a method?
```suggestion
    room_2d.remove_duplicate_vertices()
```

Reply:
lmao. I'm making a coffee; I should not have made that mistake